scripts/generate/clean-up.ts

```diff
@@ -10,6 +10,7 @@ export function cleanUp(type: string) {
   delete json.scripts.generate;
   delete json.scripts['clean-up'];
   delete json.devDependencies['@types/yargs'];
+  delete json.devDependencies['@types/fs-extra'];
```

Comment:
I think your fix is better. I missed this in previous PR. Can you rebase this PR please? 🙇